sigs/sig-security/README.md

```diff
@@ -0,0 +1,21 @@
+# Argo Sig Security
+
+Argo Sig Security is focused on improving security across the Argo project, promoting best practices, and ensuring a high level of security for end users.
+
+## Meetings
+Sig security meetings include public and non-public sections and are held twice monthly. Meetings can be found in the [Argo Project public calendar](https://calendar.google.com/calendar/embed?src=argoproj%40gmail.com&ctz=America%2FDenver).
+
+## Security Policy and Reporting Issues
+Argo Project security policy and instructions for reporting issues is availabile [here](https://github.com/argoproj/argoproj/blob/master/SECURITY.md).
+
```

Comment:
```suggestion
## Membership Requirements
- Must be an Argo project maintainer from the current list (https://github.com/argoproj/argoproj/blob/master/MAINTAINERS.md).
- Must have shown competency in discovering, solving, and communicating in the project security context.
- Must have shown a proven record of solid coding development.
- Must represent the open-source community’s best interest with a vendor-neutral practice.
- The membership will be revisited every calendar year.
```

Reply:
The team agrees on the maintainer requirement.

Reply:
Line 13/14: replace to "The maintainers to decide the membership." "Define a process for adding/removing."